### PR TITLE
[docs] Update comment about GitHub breaks

### PR DESCRIPTION
**Pull request by julien-c**
GitHub does not, in fact, appear to break on a single line.

### CODE REVIEW

**Comment by UziTech** [comment]
these lines are separated
by a single new line (`\n`)

GitHub does break on a single new line.

**Comment by UziTech** [comment]
Can you provide an example of when GitHub does not break on a single new line character?

**Comment by julien-c** [comment]
Ah, interesting, this seems to be specific to comments. GitHub does not break on a single \n in (at least):

- markdown files rendered: https://github.com/julien-c/Smellington/blob/master/README.md
- gists: https://gist.github.com/julien-c/0e6b96e77dcac99aa7412dc45ed865fb

File: docs/USING_ADVANCED.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ console.log(marked(markdownString));
 |Member      |Type      |Default  |Since    |Notes         |
 |:-----------|:---------|:--------|:--------|:-------------|
 |baseUrl     |`string`  |`null`   |0.3.9    |A prefix url for any relative link. |
-|breaks      |`boolean` |`false`  |v0.2.7   |If true, add `<br>` on a single line break (copies GitHub). Requires `gfm` be `true`.|
+|breaks      |`boolean` |`false`  |v0.2.7   |If true, add `<br>` on a single line break. Requires `gfm` be `true`.|
```
Comment (styfle) [inline docs/USING_ADVANCED.md]:
```suggestion
|breaks      |`boolean` |`false`  |v0.2.7   |If true, add `<br>` on a single line break (copies GitHub comments). Requires `gfm` be `true`.|
```

File: docs/USING_ADVANCED.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ console.log(marked(markdownString));
 |Member      |Type      |Default  |Since    |Notes         |
 |:-----------|:---------|:--------|:--------|:-------------|
 |baseUrl     |`string`  |`null`   |0.3.9    |A prefix url for any relative link. |
-|breaks      |`boolean` |`false`  |v0.2.7   |If true, add `<br>` on a single line break (copies GitHub). Requires `gfm` be `true`.|
+|breaks      |`boolean` |`false`  |v0.2.7   |If true, add `<br>` on a single line break (copies GitHub comments). Requires `gfm` be `true`.|
```
Comment (julien-c) [inline docs/USING_ADVANCED.md]:
```suggestion
|breaks      |`boolean` |`false`  |v0.2.7   |If true, add `<br>` on a single line break (copies GitHub behavior on comments, but not on rendered markdown files). Requires `gfm` be `true`.|
```

**Comment by julien-c** [comment]
Thanks for the suggestion @UziTech @styfle, I reworded the description. What do you think?

**Pull request merged**